### ISSUE TITLE
Allow wide and full alignments for blocks in widget areas

### ISSUE THREAD

**Issue by coreyworrell** [WordPress/gutenberg#35468]
## What problem does this address?

Blocks are not able to be set to wide or full alignment when inserting into a widget/sidebar area. This can be useful for sidebar areas that are used for "below content" (ie: a wide three column block for ads) or footer purposes.

## What is your proposed solution?

Possibly add parameters to `register_sidebar` to allow for these alignments, that are read by Gutenberg through REST API.
Discussion: https://wordpress.slack.com/archives/C01D71823PB/p1633425100038500

**Comment by noisysocks** [comment]
Good idea. Just need to figure out a nice place to register the alignments, as you say 🙂
